common/src/stack/command/stack/commands/add/firmware/model/plugin_basic.py

```diff
@@ -0,0 +1,81 @@
+# @copyright@
+# Copyright (c) 2006 - 2018 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+#
+# @rocks@
+# Copyright (c) 2000 - 2010 The Regents of the University of California
+# All rights reserved. Rocks(r) v5.4 www.rocksclusters.org
+# https://github.com/Teradata/stacki/blob/master/LICENSE-ROCKS.txt
+# @rocks@
+
+import stack.commands
+from stack.exception import ArgRequired, CommandError, ParamRequired
+
+class Plugin(stack.commands.Plugin):
+	"""Attempts to add all provided model names to the database associated with the given make."""
+
+	def provides(self):
+		return 'basic'
+
+	def run(self, args):
+		params, args = args
+		# Require at least one model name
+		if not args:
+			raise ArgRequired(cmd = self.owner, arg = 'model')
+
+		make, = self.owner.fillParams(
+			names = [('make', None)],
+			params = params
+		)
```

Comment:
Don't think you need keyword args in `fillParams`. I haven't see it elsewhere in our code.